proto/umee/leverage/v1/events.proto

```diff
@@ -97,6 +97,10 @@ message EventReservesExhausted {
   string borrower = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   // Outstanding bad debt
   cosmos.base.v1beta1.Coin outstanding_debt = 2 [(gogoproto.nullable) = false];
+  // Module balance remaining
+  cosmos.base.v1beta1.Coin module_balance = 3 [(gogoproto.nullable) = false];
+  // Reserves remaining
+  cosmos.base.v1beta1.Coin reserves = 4 [(gogoproto.nullable) = false];
```

Comment:
Why do we need it in an event?

Reply:
Might be useful to track if the system has become unhealthy, and not accessible through any Msg or Response or Query.